### PR TITLE
Tutorial intermediated settlement

### PR DESCRIPTION
**Pull request by johan-da**
Fixes https://github.com/digital-asset/daml-finance/issues/730

### CODE REVIEW

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Intermediated.daml
+  :language: daml
+  :start-after: -- WRAPPED_TRANSFERS_SETUP_BEGIN
+  :end-before: -- WRAPPED_TRANSFERS_SETUP_END
+
+The below diagram illustrates the setup, where edges represent accounts and stars (*) denote
+holdings::
+
+  --
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst:31]: You can wrap it into a code sample, if probably shows better

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+########################
+
+This tutorial expands upon the principles discussed in our previous
+:doc:`Internal Settlement <internal>` tutorial, providing an in-depth exploration of intermediated
+settlement involving a multi-level account hierarchy across different custodians.
+
+Understanding Intermediated Settlement with Examples
+****************************************************
+
+This tutorial features two exemplary scripts, ``runWrappedTransfersSettlement`` and
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst:14]:
```suggestion
This tutorial features two example scripts, ``runWrappedTransfersSettlement`` and
```

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+  :end-before: -- WRAPPED_TRANSFERS_DISCOVER_END
+
+These steps are converted into a batch and instructions:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Intermediated.daml
+  :language: daml
+  :start-after: -- WRAPPED_TRANSFERS_INSTRUCT_START
+  :end-before: -- WRAPPED_TRANSFERS_INSTRUCT_END
+
+The following diagram visualizes this pre- and post-settlement of the batch, where > signifies
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst:57]:
```suggestion
The following diagram visualizes this pre- and post-settlement of the batch, where ``>`` signifies
```

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+settlement instructions, and stars (*) represents the holdings::
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst:58]:
```suggestion
settlement instructions, and stars (``*``) represents the holdings::
```

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+instrument issued by the Central Bank, and setting up an account hierarchy rooted in the Central
+Bank. This hierarchy includes two custodian banks, Bank1 and Bank2, and their respective clients
+Alice, Bob, and Charlie. Holdings are created for Alice@Bank1, Bank1@CentralBank, and Charlie@Bank2:
+
+.. literalinclude:: ../../finance-settlement/daml/Scripts/Intermediated.daml
+  :language: daml
+  :start-after: -- WRAPPED_TRANSFERS_SETUP_BEGIN
+  :end-before: -- WRAPPED_TRANSFERS_SETUP_END
+
+The below diagram illustrates the setup, where edges represent accounts and stars (*) denote
```
Comment (matteolimberto-da) [inline docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst:28]:
```suggestion
The below diagram illustrates the setup, where edges represent accounts and stars (``*``) denote
```

**Review by matteolimberto-da — APPROVED** [review]
Thanks, left a few suggestions. 

I wonder whether it would be simpler to not use `Charlie` at all and do Alice -> Bob across different banks

File: docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst (hunk 1 above)
Comment (johan-da) [reply docs/2.7.0/docs/daml-finance/tutorials/settlement/intermediated.rst]: thanks, I wrapped it in a .. code-block:: none

**Comment by johan-da** [comment]
> I wonder whether it would be simpler to not use `Charlie` at all and do Alice -> Bob across different banks

let's consider that in a future  version. I already used Charlie in the internal settlement story, and tried to extend those examples step-by-step for a multi-level account hierarchie.

**Pull request merged**